### DISCUSSION TITLE
How to run a choco step only when getting a TAG

### DISCUSSION THREAD

**Discussion by adriens** [crazy-max/ghaction-chocolatey#164] [Q&A] [answered]
# Context

I'm porting a Chcolatey build project from Appveyor to github action. On the Appveyor project, I run the release only when getting a tag.
I'd like to port this to Github Actions

# Action

Could you please provide some `yaml` samples for newbees like me so they could run some choco commands only when gettin a tag ?

Thank you in advance for you help and your Github action. 🙏

**Comment by crazy-max** [comment, marked as answer]
@adriens Sorry I don't bring support directly related to GitHub Actions. I suggest to check their [quick start guide](https://docs.github.com/en/actions/quickstart). But I can give you a small hint:

```yaml
on:
  push:
    branches:
      - 'master'
    tags:
      - 'v*'
```

**Comment by adriens** [comment]
LO, yes, thanks, that's a good hint. I was about to use the `if` (and use refs) statement but your approche looks cleaner 👍
